### ISSUE TITLE
No access for the title in my custom renderTitle Method. Possible solution..

### ISSUE THREAD

**Issue by mafiusu** [testshallpass/react-native-dropdownalert#124]
#### Short Description

The Method renderTitle() in DropdownAlert.js should call the renderTitle method from the props (if exists) with the actual Title to display. Actually the method pass (this.props)

See actual method

```
  renderTitle() {
    if (this.props.renderTitle) {
      return this.props.renderTitle(this.props);
    } else {
      return (
        <Label style={StyleSheet.flatten(this.props.titleStyle)} numberOfLines={this.props.titleNumOfLines} text={this.state.title} />
      );
    }
  }
```

But shouldn't it be the following?

```
  renderTitle() {
    if (this.props.renderTitle) {
      return this.props.renderTitle(this.state.title);
    } else {
      return (
        <Label style={StyleSheet.flatten(this.props.titleStyle)} numberOfLines={this.props.titleNumOfLines} text={this.state.title} />
      );
    }
  }

```

**Comment by testshallpass** [comment]
Hi @mafiusu It's a good question. I think both should be passed down:
```return this.props.renderTitle(this.props, this.state.title);```

**Comment by msevestre** [comment]
What about you pass this.props and this.state as parameter?
The same problem also for renderMessage BTW

**Comment by testshallpass** [comment]
Fixed in v3.7.1. `this.state` is passed down in renderTitle and renderMessage prop functions.

**Comment by msevestre** [comment]
@testshallpass Awesome.
